### ISSUE TITLE
[Bug] Angular tabbed navigation item can be selected even when disabled

### ISSUE THREAD

**Issue by waterskier2007** [sparkdesignsystem/spark-design-system#1176]
## Describe The Bug
If a tabbed navigation item is disabled, it can still be selected when using arrow keys

## To Reproduce

1. Create a tabbed navigation per documentation
2. Set the third (or any) tab disabled with [disabled]="true"
3. Cycle through the tabs using the arrow keys
4. All tabs are selectable

## Expected Behavior
The third (or whichever) tab should be disabled

## Screenshots
Third tab renders as disabled as indicated by gray color (and is not clickable)
![image](https://user-images.githubusercontent.com/2089840/54231348-be70c200-44de-11e9-89a2-714135e2fd97.png)

But it can be navigated to with arrow keys
![image](https://user-images.githubusercontent.com/2089840/54231306-ad27b580-44de-11e9-906d-f09f3631480e.png)

## Desktop (please complete the following information):
 - OS: windows
 - Browser: chrome
 - Version: 72

## Additional Context
Here is the code I used to test

```
<sprk-tabbed-navigation idString="tabs-1">
        <button sprkTabbedNavigationTab analyticsString="Tab: 1" data-id="tab-1">Tab 1</button>
        <button [defaultActive]=true sprkTabbedNavigationTab data-id="tab-2">Tab 2</button>
        <button sprkTabbedNavigationTab data-id="tab-3" [disabled]="true">Tab 3</button>
        <div sprkTabbedNavigationPanel>
            <p>Tab 1 Content Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed quis rhoncus ipsum. Nulla euismod nisi est, vel consequat ante consectetur in. Ut interdum dictum est at ornare. Nam nec dapibus nibh. Integer venenatis ex eu mi euismod, non ultricies lacus venenatis.</p>
        </div>
        <div [defaultActive]="true" sprkTabbedNavigationPanel>
            <p>Tab 2 Content Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed quis rhoncus ipsum. Nulla euismod nisi est, vel consequat ante consectetur in. Ut interdum dictum est at ornare. Nam nec dapibus nibh. Integer venenatis ex eu mi euismod, non ultricies lacus venenatis.</p>
        </div>
        <div sprkTabbedNavigationPanel>
            <p>Tab 3 Content Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed quis rhoncus ipsum. Nulla euismod nisi est, vel consequat ante consectetur in. Ut interdum dictum est at ornare. Nam nec dapibus nibh. Integer venenatis ex eu mi euismod, non ultricies lacus venenatis.</p>
        </div>
    </sprk-tabbed-navigation>
```

**Comment by ssabo1** [comment]
Fixed in the same PR: Tab component ignores disabled attribute when doing Home/End key navigation
